ro.go.hmlendea.Duolingo.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: ro.go.hmlendea.Duolingo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+separate-locales: false
+command: duolingo-desktop
+
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+
+modules:
+  - name: duolingo-desktop
+    buildsystem: simple
+    build-commands:
+      - unzip -d duolingo-desktop duolingo-desktop_*.zip
+      - cp -r duolingo-desktop /app
+      - install -Dm 644 icon.svg ${FLATPAK_DEST}/share/icons/hicolor/512x512/apps/duolingo.svg
+      - install -Dm 644 duolingo-desktop/ro.go.hmlendea.Duolingo.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+      - install -Dm 644 ro.go.hmlendea.Duolingo.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm 755 duolingo-desktop.sh /app/bin/duolingo-desktop
+    sources:
+      - type: file
+        only-arches:
+          - x86_64
+        url: https://github.com/hmlendea/duolingo-desktop/releases/download/v2.0.10/duolingo-desktop_2.0.10_linux.zip
+        sha256: b9f9b09b1f79a847252dc0fdec323dffe2f8193059da81e2eae2a344c4e7f326
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/hmlendea/duolingo-desktop/releases/latest
+          version-query: .tag_name | sub("^v"; "")
+          url-query: .assets[] | select(.name=="duolingo-desktop_" + $version +
+            "_linux.zip") | .browser_download_url
+      - type: file
+        path: ro.go.hmlendea.Duolingo.metainfo.xml
+      - type: file
+        path: ro.go.hmlendea.Duolingo.png
+      - type: file
+        url: https://design.duolingo.com/86230c9ad10d9f08b785.svg
```

Comment:
you should add a sha256.

Also this look like "borrowed" from duolingo which isn't really fair use so it's probably not a good idea.

Reply:
As far as I understood from their page, that icon (along with others) are redistributable.

But just to be safe I'll just use the upstream app icon instead.

I've added the checksum as well.